libs/components/forms/src/lib/modules/field-group/field-group.component.html

```diff
@@ -17,8 +17,9 @@ <h4 *ngIf="headingLevel === 4" [class]="headingClass">
       </h4>
     </legend>
     <sky-help-inline
-      *ngIf="helpPopoverContent && labelText"
+      *ngIf="helpPopoverContent || helpKey"
```

Comment:
is this intentional? we don't want to require labelText to use help? Are we consistent about that everywhere?

Reply:
in this specific case `labelText` will always be defined since it's a required field of the new component